### PR TITLE
feat: fixed some css code and maintained iframe responsiveness

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]

File: CSS/base/reset.css [hunk 1]
```diff
@@ -87,7 +89,7 @@ button {
 }
 
 .headings {
-    margin: 1rem 0rem 0rem 0rem;
+    margin: 0.5rem 0rem 0rem 0rem;
```
Comment (fahd23) [inline CSS/base/reset.css:92]: You can give like margin:5rem 0 0;

File: CSS/button.css [hunk 2]
```diff
@@ -1,34 +1,26 @@
...
-.button-container {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: center;
-}
-
 .default-button-container {
     border: 1px solid var(--dark-primary);
     padding: 2rem 10% 2rem 10%;
```
Comment (fahd23) [inline CSS/button.css]: Can you tell me, Why you use % for padding here?

File: style.css [hunk 3]
```diff
@@ -70,12 +65,16 @@ body {
 }
 
 .landing-page-paragraph {
+    background-color: #000001ab;
```
Comment (fahd23) [inline style.css]: Use CSS variable for consistency

**Review by fahd23 — APPROVED** [review]
Given some suggestions, Good to merge

File: CSS/button.css (hunk 2 above)
Comment (ajit-sharmaGH) [reply CSS/button.css]: for responsiveness, I want the padding should decrease or increase side by side as per screen sizes! I don't know it is a good practice or not!

File: style.css (hunk 3 above)
Comment (ajit-sharmaGH) [reply style.css]: Uff, I missed that one !

**Pull request merged**